backend/control_center/notification/ui_views.py

```
@@ -0,0 +1,55 @@
+
+    def get_queryset(self):
+        qs = Notification.objects.filter(user=self.request.user).order_by("-created_at")
+        read = self.request.query_params.get("read")
+        if read is not None:
+            if read.lower() == "true":
+                qs = qs.filter(is_read=True)
+            elif read.lower() == "false":
+                qs = qs.filter(is_read=False)
+        return qs
+
+    def partial_update(self, request, *args, **kwargs):
```

Ruff (0.13.3)

Unused method argument: `args`

(ARG002)